src/views/events/BoardView.vue

```diff
@@ -2,9 +2,14 @@
   <div class="tile is-ancestor">
     <div class="tile is-parent">
       <div class="tile is-child">
-        <div class="title">Board-view</div>
+        <div class="title">Board view</div>
 
-        <div class="field" v-if="boardBodies.length > 0">
+        <div class="field" v-if="boardBodies.length === 1">
+          <label>You are currently viewing applications from </label>
+          <strong v-for="body in boardBodies" v-bind:key="body.id">{{ body.name }}</strong>.
```

Comment:
you can also take the 1st one and avoid `for` loop there

Reply:
Yes very true. I did not know you can directly access your data in the template like that. So also learned something new again today 😄